src/melee.cpp

```diff
@@ -1204,14 +1206,20 @@ bool player::can_weapon_block() const
 {
     return (weapon.has_technique( WBLOCK_1 ) ||
             weapon.has_technique( WBLOCK_2 ) ||
-            weapon.has_technique( WBLOCK_3 ));
+            weapon.has_technique( WBLOCK_3 ) ||
+            weapon.has_technique( WBLOCK_4 ) ||
+            weapon.has_technique( WBLOCK_5 ));
```

Comment:
It appears this might be easier than I first thought. I first thought you meant to rework `blocking_ability`, but `can_weapon_block` should be simpler.